wix-pricing-plans-v2/wix-pricing-plans-v2/Orders.service.json

```diff
@@ -368,26 +979,262 @@
         ],
         "examples": [
           {
-            "title": "managementGetOrder example",
+            "title": "Get an order  ",
```

Comment:
Is it possible to give the examples a title that includes management in it? I'm honestly not sure what the "management" part means, but I feel like it might be helpful to include that for the reader's understanding.

Take this as a suggestion, not as a must-fix.

Reply:
This is actually a good point to bring up.. some of these functions are questionably named... `managementListOrders`, `managementGetOrder`, `memberListOrders`, `memberGetOrder`... I think that all management means here is with elevated permissions and get all plans, versus memberListOrders, which is only listing orders for THAT logged in member, and does not require elevated permission. Do you have any suggestions based on that info?